terraform.tf

```diff
@@ -94,6 +94,9 @@ module "ecs" {
   send_job_alerts_daily_email_command  = "${var.send_job_alerts_daily_email_command}"
   send_job_alerts_daily_email_schedule = "${var.send_job_alerts_daily_email_schedule}"
 
+  send_expiry_alerts_schedule = "${var.send_expiry_alerts_schedule}"
+  send_expiry_alerts_command = "${var.send_expiry_alerts_command}"
```

Comment:
Nitpick: please run `terraform fmt` on this (`Format Document` does this in VSCode if you have the Terraform extension).